docs/observability/index.asciidoc

```diff
@@ -0,0 +1,24 @@
+[chapter]
+[role="xpack"]
+[[observability]]
+= Observability
+
+Observability in {kib} enables you to add and monitor your logs, system
+metrics, uptime data, and application traces, as a single stack. 
+
+On the *Observability Overview* page you have:
+
+* A central place to add and configure your data sources.
+* A variety of charts displaying analytics relating to each data source.
+* *View in app* options to drill-down and anaylze data in the {logs-app}, {metrics-app}, {uptime-app}, and the APM app. 
+* An alerts chart to keep you informed of any issues that you may need to resolve quickly.
+
+[role="screenshot"]
+image::observability/images/observability-overview.png[Observability Overview in {kib}]
+
+[float]
+== Get started
+
+{kib} provides step-by-step instructions to help you add and configure your data
+sources. The {observability-guide}/index.html[Observability Guide] is a good source for more detailed information
```

Comment:
{observability-guide}/index.html[Observability Guide] isn't appearing as a link

Reply:
@KOTungseth That's strange. It looks fine here in the preview: https://kibana_73041.docs-preview.app.elstc.co/guide/en/kibana/master/observability.html